src/libs/location.c

```diff
@@ -616,15 +616,15 @@ static void _lib_location_parser_start_element(GMarkupParseContext *cxt, const c
       {
 
         if(strcmp(*avalue, "village") == 0)
-          place->type = LOCATION_TYPE_RESIDENTAL;
+          place->type = LOCATION_TYPE_RESIDENTIAL;
         else if(strcmp(*avalue, "hamlet") == 0)
           place->type = LOCATION_TYPE_HAMLET;
         else if(strcmp(*avalue, "city") == 0)
           place->type = LOCATION_TYPE_CITY;
         else if(strcmp(*avalue, "administrative") == 0)
           place->type = LOCATION_TYPE_ADMINISTRATIVE;
-        else if(strcmp(*avalue, "residental") == 0)
-          place->type = LOCATION_TYPE_RESIDENTAL;
+        else if(strcmp(*avalue, "residential") == 0)
+          place->type = LOCATION_TYPE_RESIDENTIAL;
```

Comment:
I believe there needs to be code here that can digest both `residental` (backward compatible) and `residential`

Reply:
Good point indeed! I missed that, Can you propose a PR?

Reply:
@luzpaz : I think this even need to be reverted. The term "residental" exists and seems to come from an external library. So better revert all RESIDENTIAL back to RESIDENTAL and wait for @phweyland comment. TIA.

Reply:
`residental` seems to be discouraged by openstreetmap but the tag exists anyway (16692 times):
![image](https://user-images.githubusercontent.com/23012047/154489512-c472bc35-2a4e-4e78-a6fb-c822eaf3c9c0.png)
Map creators should use `residential`;
So the above @luzpaz suggestion sounds good.
Thanks

Reply:
please advice on how to proceed? (if needed?)

Reply:
Given @phweyland answer I would add a check for both `"residential"` and `"residental"` as alias and set `place->type = LOCATION_TYPE_RESIDENTIAL;`.

Reply:
See #11168